docker-compose.yml

```diff
@@ -9,6 +9,7 @@ services:
     environment:
       - "DJANGO_SETTINGS_MODULE=modernomad.settings.local"
       - "DATABASE_URL=postgres://postgres@postgres/postgres"
+      # Read these envvars from .env
```

Comment:
Clarify that these are being taken either from `.env` or the environment it was started in, which may make a difference.